Dockerfile.cli

```
@@ -0,0 +1,29 @@
+FROM node:10-alpine
+
+WORKDIR /gatsby
+
+ARG GATSBY_VERSION=^2.0.0
+ARG VIPS_VERSION=8.6.3-r0
```

Building the Dockerfile.cli fails with:

```
+ apk add --no-cache 'vips-dev=8.6.3-r0' --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing
fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  vips-dev-8.7.0-r0:
    breaks: world[vips-dev=8.6.3-r0]
  pc:fftw3 (missing):
    required by: vips-dev-8.7.0-r0[pc:fftw3]
                 vips-dev-8.7.0-r0[pc:fftw3]
                 vips-dev-8.7.0-r0[pc:fftw3]
The command '/bin/sh -c set -x   && apk add --no-cache vips-dev=${VIPS_VERSION}   --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing   && apk add --no-cache   fftw-dev   g++   git   make   python2   binutils   git' returned a non-zero code: 2
```

I don't entirely understand why the error is caused for asking for an older version but using the version stated(or not requesting a version?) will avoid the `unsatisfiable constraints` error and let the image build properly:

```suggestion
ARG VIPS_VERSION=8.7.0-r0
```